src/cljs/athens/page.cljs

```diff
@@ -68,33 +82,39 @@
                             ^{:key uid}
                             [:span
                              {:style {:cursor "pointer"}
-                              :on-click #(dispatch [:navigate :page {:id uid}])}
+                              :on-click #(on-block-click uid)}
                              (or string title)]))
                         @parents))]
-       [:h2 {:style {:margin 0}} (str "• " (:block/string @node))]
-       [:div {:style {:margin-left 20}}
+       [:div
+        {:style {:margin 0}
+         :content-editable true}
```

Comment:
Sounds like content-editable is a much easier/better solution than `textarea`. Why then do you think Roam uses `textarea`?

Reply:
There's two potential reasons, one I think is more likely than the other. The first, unlikely, reason is that textarea is a form element so using content-editable means having to manually handle submission (seems trivial using on blur). The second, more likely reason, is that content-editable preserves the styling unlike textarea, so I'm thinking maybe they tried content-editable first but were not able to implement their custom styling syntax with the way content-editable works. We won't know until we try.

Reply:
There's two potential reasons, one I think is more likely than the other. The first, unlikely, reason is that textarea is a form element so using content-editable means having to manually handle submission (seems trivial using on blur). The second, more likely reason, is that content-editable preserves the styling unlike textarea, so I'm thinking maybe they tried content-editable first but were not able to implement their custom styling syntax with the way content-editable works. Or they just wanted the user to see plain unformatted text when they are editing and it is a design decision rather than a technical one. We won't know until we reach that stage.